### ISSUE TITLE
intake-stac setup and error on aws search example

### ISSUE THREAD

**Issue by dogrdon-cu** [intake/intake-stac#64]
Hello, 

I've noticed that there is an error running the [aws search notebook example](https://github.com/intake/intake-stac/blob/master/examples/aws-earth-search.ipynb). Am getting, at cell 11 loading the item to Dask `da = item.B04(chunks=dict(band=1, y=2048, x=2048)).to_dask()` the following error: 

`ValueError: No plugins loaded for this entry: image/tiff; application=geotiff; profile=cloud-optimized
A listing of installable plugins can be found at https://intake.readthedocs.io/en/latest/plugin-directory.html .`

I've ensured that the module is installed and the example on the intake-stac readme does run fine. 

Initially was running this inside of the [pangeo notebook docker image](https://github.com/pangeo-data/pangeo-docker-images/tree/master/pangeo-notebook) and thought that might be the issue but am getting this same error just installing the dependencies in a clean virtual environment.

**Comment by scottyhq** [comment]
Hi @dogrdon-cu, sorry about that, we're working on a fix over in https://github.com/intake/intake-stac/pull/61 and hope to have a new release ASAP that fixes this.

In the meantime a workaround is to pin `intake-xarray=0.3.1` in your environment

**Comment by dogrdon-cu** [comment]
thanks @scottyhq that worked. 

FWIW, I noted that I also had to remove and reinstall `intake-stac` with 
`pip uninstall intake-stac`
`pip install git+https://github.com/intake/intake-stac`
in the [pangeo-notebook](https://hub.docker.com/r/pangeo/pangeo-notebook) to get from the plugin error to the `open_local` error ref. in #61

**Comment by hssiddiqui** [comment]
@scottyhq Any workarounds for dealing with 'sentinel-s2-l1c' products? From the example above, intake-xarray=0.3.1 works for Sentinel 2 L2A dataset but not for Sentinel 2 L1C dataset returning the following error:

ValueError: No plugins loaded for this entry: **image/jp2**
A listing of installable plugins can be found at https://intake.readthedocs.io/en/latest/plugin-directory.html .

It seems there's a plugin issue due to the different format of the datasets from Sentinel 2 even though they're both raster files that should be handled by the existing plugins.

**Comment by scottyhq** [comment]
@hssiddiqui , thanks for pointing this out. this pending PR #61 sets the driver to 'rasterio' by default so even if the entry type is not recognized we'll attempt to read the data rather than going straight to the ValueError.

So for now you'll have to install from the branch in the PR above. Hopefully this week we'll get new versions released...

Note also the S2L1C imagery is in a requester-pays bucket, so you'll have to use the following syntax:
```python
import rasterio
awsconfig = rasterio.session.AWSSession(requester_pays=True)
with rasterio.Env(session=awsconfig):
    da = item.B01.to_dask()
```

**Comment by scottyhq** [comment]
closed by #61
